src/components/exercise-table/Exercise-table.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+// Define the Set type
+interface Set {
+  reps: number;
+  weight: number;
+}
+
+// Define the Exercise type
+interface ExerciseEntry extends Exercise { 
+  sets: Set[];
+}
+  id: number;
+  name: string;
+  label: string;
+  bodyPart: string;
+  sets: Set[];
+}
+
+interface ExerciseTableProps {
+  exercise: Exercise;
+  handleSetChange: (
+    index: number,
+    setIndex: number,
+    field: string,
+    value: string
+  ) => void;
+  handleAddSet: () => void;
+}
+
+const ExerciseTable: React.FC<ExerciseTableProps> = ({
```

Comment:
Are we using ExerciseTable somewhere in the code? I can't find where it's being used 🤔

Reply:
Oh you can ignore this file, I was attempting to make the table a separate reusable component for each exercise that is added, but that'll probably be done in another ticket. Should I just delete this file for the purpose of this pull request?

Reply:
Sounds good - yeah delete :)